src/common/transformations/src/transformations/common_optimizations/transpose_sinking_split.cpp

```diff
@@ -76,6 +76,22 @@ bool IsSplitSinked(const Output<Node>& output) {
     return HasInputSplitAndTransposeSiblings(output) && is_sinking_node(output);
 }
 
+bool GetSplitAxis(const std::shared_ptr<Constant>& split_axis, const ov::Rank& rank, int64_t& axis) {
```

Comment:
I think it is better to generalize this function to execute normalization of `axis` for Concat and Split, etc.

Reply:
normalization of axis for Concat op is already implemented as part of validate_and_infer_type method of Concat
if we call concat_node->get_concatenation_axis(); method, it should return normalized axis already